state-chain/runtime/src/runtime_apis.rs

```diff
@@ -197,6 +197,7 @@ pub struct BrokerInfo {
 	pub btc_vault_deposit_address: Option<String>,
 	pub affiliates: Vec<(AffiliateShortId, AccountId32)>,
 	pub bond: AssetAmount,
+	pub affiliate_details: Vec<(AffiliateDetails, AssetAmount)>,
```

Comment:
Unfortunately this is a breaking change to the runtime API. We can do this but we should manage it using runtime api versioning (see the doc comment on the CustomRuntimeApi trait).

Alternatively maybe we can get the data we need via `get_affiliates`.